pandas/tests/io/formats/test_format.py

```diff
@@ -1047,6 +1047,30 @@ def test_to_string_with_col_space(self):
         no_header = df.to_string(col_space=20, header=False)
         assert len(with_header_row1) == len(no_header)
 
+    def test_to_string_with_column_specific_col_space(self):
+        df = DataFrame(np.random.random(size=(3, 3)), columns=["a", "b", "c"])
+
+        msg = (
```

Comment:
Can you split all of the error testing into another test called `test_to_string_with_column_specific_col_space_raises`? We use that convention often throughout the tests